index.html

```diff
@@ -150,7 +154,7 @@ <h2>Multi-PostStories</h2>
         </section>
 
         <section class="primary-font about-me">
-            <div class="">
+            <div>
```

Comment:
- [ ] your website looks awesome and you did a great job 👏 , kindly fix these minor issues so it becomes ready to be approved 💯 : 

- Kindly add the ' Lets connect ' title before the social links in the about me section 
- Kindly add more space on top and below the about me content (padding will be great) 🙏 

![image](https://user-images.githubusercontent.com/89282221/228858955-82d5cbfd-9f7a-424d-a8da-80d61ca1b812.png)